planetarium/NineChronicles
# Prevent gear unequipped after showing `FriendInfoPopup`.

### Description

SSIA

### How to test

1. Open arena.
2. Inspect any other player's gear.
3. Play `RankingBattle`.
4. Check if gears is not unequipped.

### Related Links
https://app.asana.com/0/1141562434100787/1200419177484339
https://app.asana.com/0/1141562434100787/1200503122421394

### Required Reviewers

@planetarium/9c-dev
